ramls/loan.json

```diff
@@ -21,6 +21,11 @@
       "description": "ID of the item lent to the patron",
       "type": "string"
     },
+    "itemEffectiveLocationAtCheckOut": {
```

Comment:
Is the intention to only store the effective location, or could there be other aspects of the location also stored at check out?

Reply:
Right now, only the effective location.

Reply:
Ok, thanks